src/main/groovy/pl/allegro/tech/build/axion/release/domain/VersionConfig.groovy

```diff
@@ -49,6 +49,8 @@ class VersionConfig {
 
     HooksConfig hooks = new HooksConfig()
 
+    List<String> foldersToExclude = []
```

Comment:
i wonder if we should use "directory" instead of folder (`directory` is already used in `RepositoryConfig`), so maybe `dirsToExclude`?

Reply:
Agreed.